src/librustdoc/html/layout.rs

```diff
@@ -55,6 +55,13 @@ crate fn render<T: Print, S: Print>(
     <link rel=\"stylesheet\" type=\"text/css\" href=\"{static_root_path}normalize{suffix}.css\">\
     <link rel=\"stylesheet\" type=\"text/css\" href=\"{static_root_path}rustdoc{suffix}.css\" \
           id=\"mainThemeStyle\">\
+    <link rel=\"preload\" as=\"font\" crossorigin=\"anonymous\" href=\"{static_root_path}FiraSans-Regular.woff2\">\
```

Comment:
I really have cold feet about this change. Another reason is that currently, the css will only load the font it needs, stopping at the first one working whereas it will download all of them in any case here. So here what I suggest: can you open a PR with this change so we can go through this carefully and get the other changes merged please? Like that, we can already add most of the improvements already (which are great and I can't wait for this faster rustdoc!).

Reply:
Done. And I agree that the `preload` version of this conflicts with the `woff` / `woff2` change I recently landed; I think a followup change would need to do this with an inline `<style>` in order to properly account for that.

Reply:
Thanks!